FAQ-Master.md

```diff
@@ -201,14 +201,33 @@ courses of action. OpenC2 can be utilized to populate STIX COA
 SDOs for sharing automated courses of action for the purpose of
 responding to cyber incidents in cyber-relevant time.
 
-### How does OpenC2 relate to [MISP](https://www.misp-project.org/features.html)? 
+### How does OpenC2 relate to [PACE](https://opencybersecurityalliance.org/pace/)?
+
+Posture Attribute Collection and Evaluation (PACE) is an
+[Open Cybersecurity Alliance](https://opencybersecurityalliance.org/)
+effort to create a
+comprehensive automated strategy for understanding security posture.
+OpenC2 plays two roles in PACE.
+PACE uses OpenC2 for command & control to obtain security posture information,
+and for the command & control of the PACE system itself
+(e.g. request securit posture evaluation of a device).
+OpenC2 also plays a role in the actions that result from posture evaluation
```

Comment:
Maybe "... plays a role in _implementing_ the actions that .."

Reply:
done